ts/src/base/functions/generic.ts

```diff
@@ -129,7 +129,7 @@ const flatten = function flatten (x: any[], out: any[] = []) {
 
 const pluck = (x: Dictionary<any>, k: any) => values (x).filter ((v) => k in v).map ((v) => v[k]);
 
-const omit = (x: Dictionary<any>, ...args) => {
+const omit = (x: Dictionary<any>, ...args: any) => {
```

Comment:
@samgermain we can't type omit properly here because we're not using it correctly, by accepting the spread operator we shouldn't do `this.omit(x, [a,b,c])` but `this.omit(x, a,b,c)` but that would not work well in other languages.

For now we can keep the `any` and in the future fix omit's usage.